e2e/tests/apikey-lifecycle.spec.ts

```
@@ -0,0 +1,347 @@
+  await page.waitForLoadState('networkidle');
+}
+
+async function navigateToMyAPIKeys(page: Page, namespace: string): Promise<void> {
+  await spaNavigate(page, `/k8s/ns/${namespace}/devportal.kuadrant.io~v1alpha1~APIKey`);
+}
+
+async function navigateToAPIKeyDetails(
+  page: Page,
+  namespace: string,
+  name: string,
+): Promise<void> {
```

ESLint

[error] Replace `⏎··page:·Page,⏎··namespace:·string,⏎··name:·string,⏎` with `page:·Page,·namespace:·string,·name:·string`

(prettier/prettier)